weblate/addons/tasks.py

```diff
@@ -139,3 +148,8 @@ def postconfigure_addon(addon_id: int, addon=None) -> None:
 @app.on_after_finalize.connect
 def setup_periodic_tasks(sender, **kwargs) -> None:
     sender.add_periodic_task(crontab(minute=45), daily_addons.s(), name="daily-addons")
+    sender.add_periodic_task(
+        crontab(hour=24),
```

Comment:
I don't think that hour 24 actually works in cron. Also, you should specify a minute so that it is not run on minute=0 to spread the load:

```suggestion
        crontab(hour=0, minute=40),
```